tests/unittests/multimodal/lerobot_tests.py

```
@@ -0,0 +1,1778 @@
+            self.assertNotIn(root, json.dumps(isolated.sample, default=str))
+
+            missing_media_type = dict(raw)
+            missing_media_type["media_reference"] = descriptor
+            missing_media_type.pop("_media_type", None)
+            isolated = asyncio.run(
+                _create_sample_item(
+                    dataset,
+                    missing_media_type,
+                    {},
+                    {},
+                    True,
```

Ruff (0.16.2)

[warning] Boolean positional value in function call

(FBT003)